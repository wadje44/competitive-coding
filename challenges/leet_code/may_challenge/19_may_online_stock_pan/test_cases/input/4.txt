[24,5,33,80,97,11,88,5,72,3,43,88,40,93,72,61,17,35,42,60,100,29,51,94,35,63,77,64,76,20,96,18,59,30,64,30,44,69,27,73,35,14,62,17,44,40,8,72,69,78,55,56,17,21,23,53,82,36,89,74,72,34,12,97,86,41,87,90,35,51,34,70,92,99,75,31,89,100,35,95,52,97,100,82,69,16,1,60,71,94,52,70,65,21,65,99,12,69,62,38]