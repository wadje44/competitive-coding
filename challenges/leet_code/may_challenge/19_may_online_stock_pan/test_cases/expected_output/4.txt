[1, 1, 3, 4, 5, 1, 2, 1, 2, 1, 2, 7, 1, 9, 1, 1, 1, 2, 3, 4, 21, 1, 2, 3, 1, 2, 3, 1, 2, 1, 10, 1, 2, 1, 4, 1, 2, 7, 1, 9, 1, 1, 3, 1, 2, 1, 1, 8, 1, 19, 1, 2, 1, 2, 3, 4, 26, 1, 28, 1, 1, 1, 1, 43, 1, 1, 3, 4, 1, 2, 1, 4, 9, 53, 1, 1, 3, 78, 1, 2, 1, 4, 83, 1, 1, 1, 1, 3, 5, 7, 1, 2, 1, 1, 3, 13, 1, 2, 1, 1]